### PR TITLE
Azure: Update job names, add cpuFuncTests

### PR DESCRIPTION
**Pull request by azhogov**

### CODE REVIEW

File: azure-pipelines.yml [hunk 1]
```diff
@@ -404,18 +226,18 @@ jobs:
...
+      rd /Q /S $(BUILD_DIR)
+      mkdir $(BUILD_DIR)\bin
       rd /Q /S dldt-build
       mkdir dldt-build
-      cd dldt-build
     displayName: 'Create build directory'
   - task: CMake@1
     inputs:
-      workingDirectory: C:\c-dldt-build
+      #cmakeArgs: -G"Visual Studio 15 2017 Win64" -DCMAKE_BUILD_TYPE=$(BUILD_TYPE) -DENABLE_TESTS=ON $(Build.Repository.LocalPath)
```
Comment (SDxKeeper) [inline azure-pipelines.yml:236]: it's better not to keep garbage

File: azure-pipelines.yml [hunk 2]
```diff
@@ -286,114 +207,15 @@ jobs:
...
-#    displayName: 'IE CAPITests'
-#    continueOnError: true
-
-# IE_Win_self
-- job: IE_Win_self
+# Win
+- job: Win
   timeoutInMinutes: 0
   pool:
+    #vmImage: 'vs2017-win2016'
```
Comment (SDxKeeper) [inline azure-pipelines.yml:214]: I think this is wrong comment

File: azure-pipelines.yml [hunk 3]
```diff
@@ -177,6 +90,10 @@ jobs:
     workingDirectory: dldt-build
     displayName: 'IE FuncTests'
     continueOnError: true
+  - script: ../bin/intel64/$(BUILD_TYPE)/cpuFuncTests
```
Comment (SDxKeeper) [inline azure-pipelines.yml:93]: it's better to fix relative paths later.

File: azure-pipelines.yml (hunk 1 above)
Comment (azhogov) [reply azure-pipelines.yml:236]: I agree, but it is under testing now... will be removed later

**Pull request merged**